c_glib/arrow-glib/basic-data-type.h

```diff
@@ -418,6 +418,22 @@ struct _GArrowTemporalDataTypeClass
   GArrowFixedWidthDataTypeClass parent_class;
 };
 
+#define GARROW_TYPE_BINARY_VIEW_DATA_TYPE (garrow_binary_view_data_type_get_type())
```

Comment:
Should I move this definition to the bottom? (Just in case)

Reply:
Yes.
`#define GARROW_TYPE_...` should be near the related `G_DECLARE_DERIVABLE_TYPE()` and so on.

Reply:
Thanks. moved to the bottom (inside `G_BEGIN_DECL`/`G_END_DECL`)